azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/SetupFragment.kt

```diff
@@ -52,7 +54,16 @@ internal class SetupFragment :
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         viewModel.init(viewLifecycleOwner.lifecycleScope)
+        toolbar = view.findViewById(R.id.azure_communication_setup_toolbar)
```

Comment:
NIT: It will be good to create view and view model for toolbar. Example: PreviewAreaView